src/portalbackend/PortalBackend.DBAccess/Extensions/UniqueIdentifiersExtension.cs

```diff
@@ -33,4 +33,16 @@ public static string GetUniqueIdentifierValue(this UniqueIdentifierId uniqueIden
             UniqueIdentifierId.EORI => "gx:eori",
             _ => throw new ArgumentOutOfRangeException(nameof(uniqueIdentifierId), uniqueIdentifierId, "Unique Identifier not found for Clearing House Conversion")
         };
+
+    public static string GetUniqueIdentifierValue(this string uniqueIdentifierValue, UniqueIdentifierId uniqueIdentifierId, string countryCode) =>
+        // Purpose of adding switch case here is to have a possibility in the future to format other identifier's values as well
+        uniqueIdentifierId switch
+        {
+            UniqueIdentifierId.COMMERCIAL_REG_NUMBER => uniqueIdentifierValue,
+            UniqueIdentifierId.VAT_ID when !uniqueIdentifierValue.Contains(countryCode) => string.Format("{0}{1}", countryCode, uniqueIdentifierValue),
```

Comment:
```suggestion
            UniqueIdentifierId.VAT_ID when !uniqueIdentifierValue.StartsWith(countryCode) => string.Format("{0}{1}", countryCode, uniqueIdentifierValue),
```
Isn't `Contains` a bit broad here? If the letters of the country code is in a different part of the identifier it will not concatenate.  Maybe StartsWith has less false positives.